src/components/ArticlePage.jsx

```diff
@@ -23,24 +24,79 @@ const ArticlePage = () => {
     setShowComments((prevShowComments) => !prevShowComments);
   };
 
+  const changeVotes = (event) => {
+    if (event.target.innerText === "△") {
```

Comment:
I think it would make for a better condition to instead invoke changeVotes with a -1 or a +1 , and then base the logic off of that instead of the innerText, because that makes your function more flexible and reusable overall